pwiz/analysis/spectrum_processing/SpectrumList_IonMobility.cpp

```diff
@@ -134,7 +143,10 @@ PWIZ_API_DECL bool SpectrumList_IonMobility::canConvertIonMobilityAndCCS(IonMobi
 PWIZ_API_DECL bool SpectrumList_IonMobility::hasCombinedIonMobility() const
 {
     if (sl_ == nullptr)
-        return false;
+    {
+        // Not a native reader, but mzML may have 3-array representation
+        return has_mzML_combined_ion_mobility_;

```

Comment:
Can't we just check equipment/units here instead of having an extra bool?

Reply:
I agree it feels clunky, but what combination would that be? Same equipment and units could have either representation in an mzML doc.

Reply:
You're right, nevermind.